### DISCUSSION TITLE
个案 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#482] [General]
# 个案 - Miao Yu | 于淼

https://yufree.cn/cn/2024/11/19/case/

**Comment by conge** [comment]
违法和犯罪不是一回事。我想你的拉美裔同事说非法移民都是罪犯，本意大概是说他们都违法了。至于违法行为是否该收到惩罚和驱逐，大概是你们真正的分歧所在。但你难道不希望进入你国家的人是通过合法渠道的来的吗？

关于，“低薪或重体力劳动的活本来就是没人干”，解决方法一定是要雇佣非法劳工吗？而且左派真的怕“抱怨人工费高”吗？我看不是的，左派一直在用立法的方式提高最低工资。且按照逻辑，非法劳工少了，企业主就得把“低薪或重体力劳动”变成不低薪的，福利好些的工作，用市场而非立法的方式去解决低薪问题，岂不更少社会摩擦？

关于变性问题，并不是只有变性手术会产生不可逆的影响，puberty blocker和Hormone therapy的长期使用同样可以，而后两者在变性群体中更为常见（1）。而且，gender identity或者是身份认同，是个复杂的问题，简化的看，是对于自身性别的看法与物质身体产生差异。通常采用的是改变想法而非改变物质的方式进行同一。而gender affirming 用相关药物和手术，改变肉身，尤其用在还在发育中的孩子，是非常不人道的。说这个罕见，是弱化问题的严重性。如果你认为gender affirming care是对的，何必强调“罕见”来弱化它呢？对于不希望孩子用药物戕害身体去应可以用心理引导的问题的家长来说，一就是多。

“大概率也会因为性别认同抑郁甚至自杀”的说法也站不住脚，此说法要是成立的话，进行过gender affirming 相关药物和手术的人群抑郁自杀率应当低于或同与大众数据。但事实并非如此，2020年一篇文章说手术降低了此人群的心理治疗的使用，但很少有人知道后来此研究发布了corrrection，收回了自己的claim（2）。

最后看来落在了选举上。那样的话，彼此说服其实没有必要。彼此说服，只在追求真相的时候，才值得。

（1）https://www.reuters.com/investigates/special-report/usa-transyouth-data/
（2）https://segm.org/ajp_correction_2020

**Reply by yufree** [reply]
首先，我对国家这个建制持怀疑态度并避免讨论国家这个概念，发生在国家间的移民与发生在城市间的移民没有本质区别，之前国内不解锁城乡身份，住在城市需要暂住证，没有会被遣送，这跟美国遣送非法移民理念一致，我认为都是会随时间被废弃的制度。至于怎么来的，程序上合法重要，但马斯克自己就钻过J签空子，我更关心这些人产生的贡献与问题，有贡献要认可，有问题就解决，纠结来源意义有限。
我不清楚咋就成了左派，我对左右之分深恶痛绝，写的只是自己的思考，根本就没提最低工资的事，不知道咋联想出来的。不过既然说到这了，低薪或重体力劳动的最佳替代是机器人而不是人，但现实是移民劳工更便宜，市场选择了新移民从事这些工作，高薪高福利的工作本来也不对他们开放。反对他们驱逐他们日本就曾经这么做，现在不也开放劳工移民了，社会摩擦这东西不是关起门来矛盾就少了，关起门来可能会暴露出更大的内部矛盾。
变性问题我不认为身份认同是需要心理引导的，这本身不算是异常，没必要用家长价值观来引导，自己怎么处理自己的身体只要自己承担后果没必要干涉。而且执行层面一年不到一百人说明实质影响人数不大，对其恐惧比这件事本身影响更大。家长如果不想孩子受到伤害，最好的方式是沟通，给出自己的判断及依据并尊重其选择，而不是一味抵制。再说现在认为的不可逆或所谓“戕害”在技术发展下可能只是临时判断，跟人道扯不上边，真正的人道不是高高在上的指导，而是互相尊重对方作为人有不同价值观下的选择自由。
“大概率也会因为性别认同抑郁甚至自杀”这个说法来源于我认识人的经历，确实可能没啥统计意义，我也没觉得社会或家庭认可很重要，但有人自己的性别选择得不到至亲认可是真的会抑郁。我理解不了，但也不会告诉他这是心理问题，是“异常”的，别人觉得自己应该是啥性别又不影响我的性别，我没资格指指点点，每个人有每个人的修行。另外你给的研究关注的是去进行心理治疗的那一部分，别忘了很多人压根不知道有这个途径。
写这些不是想说服你，很多类似问题没有唯一解，每个人能找个局部最优说服自己就不错了。

**Comment by conge** [comment]
写了很多。都删了。你说的对，不用说服。

只对马斯克的例子多说一句：马斯克也许真的处于非法滞留状态过，但他也的确为了fix这个问题，获得J-1后来H1-B。据说他是自己出境，然后换签证重新入镜，获得合法工作的身份。现在非法滞留的人如何学马斯克，自行离境并换合法的方式重新进入，倒是挺好的解决方式。

**Comment by scorpion9111** [comment]
外面生活久了,该回村里体验下高压锅的生活了
